### DISCUSSION TITLE
Logs: Cannot find module issue

### DISCUSSION THREAD

**Discussion by enesse** [vercel/vercel#8570] [Help] [answered]
We are seeing a lot of this issue in our logs: 

```
ERROR	Error: Cannot find module '/var/task/web/.next/server/pages/api/auth/[...nextauth].js'
Require stack:
/var/task/web/node_modules/next/dist/server/require.js
/var/task/web/node_modules/next/dist/server/next-server.js
/var/task/web/___next_launcher.cjs
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.requirePage (/var/task/web/node_modules/next/dist/server/require.js:47:12)
    at Object.loadComponents (/var/task/web/node_modules/next/dist/server/load-components.js:56:23)
    at NextNodeServer.findPageComponents (/var/task/web/node_modules/next/dist/server/next-server.js:442:63)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async NextNodeServer.renderToResponse (/var/task/web/node_modules/next/dist/server/base-server.js:1079:48) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/var/task/web/node_modules/next/dist/server/require.js',
    '/var/task/web/node_modules/next/dist/server/next-server.js',
    '/var/task/web/___next_launcher.cjs'
  ],
  page: '/api/auth/callback/cognito'
}
```
No issues have been reported from end-users and Next Auth seems to be working fine when testing the site, but still, we would like to get it fixed. Does anyone know what could be causing this issue?

**This has been cross-posted on the Next Auth Github board as well**
https://github.com/nextauthjs/next-auth/issues/5353

**Comment by amyegan** [comment]
Hi @enesse! Sorry you're running into errors, and thank you for sharing the link to your `next-auth` discussion. 

Can you share more about what you're seeing? For example, when does this error appear and how often? If you can share a deployment id or URL, that will help me know where to look in the logs for additional information. Sharing a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) (or your repo if it is public) will also help me find the source of the problem.

**Reply by enesse** [reply]
Hi @amyegan, we've only noticed the issue in our logs. As far as I know no users have reported this error, yet. It doesn't happen very often either. Last week https://www.lawstudies.com/ had about 2 million requests per day and the above mentioned issue occurred 50 times per day. The repo is unfortunately not open to the public, no.

**Reply by amyegan** [reply]
Since you are not able to share a reproduction, can you share what versions of Next and NextAuth you are using? Are they pinned to specific versions in your package.json or lockfile?

**Reply by enesse** [reply]
At the time of errors: next 12.1.4 and next-auth 4.10.2, but after upgrading both packages (and upgrading to react 18) we no longer see this issue in our logs. Thanks anyway, @amyegan.

**Reply by amyegan** [reply]
There was a bug related to your issue that was fixed in more recent versions. That's why I asked about version numbers 😅 

I'm glad the upgrade worked for you!

**Comment by enesse** [comment, marked as answer]
Upgrading all packages seems to have fixed the issue.
